extensions/empty-screenshots/package.json

```diff
@@ -50,12 +75,21 @@
               "value": "90"
             }
           ]
+        },
+        {
+          "name": "prefixString",
+          "type": "textfield",
+          "title": "Custom prefix (optional)",
+          "required": false,
+          "description": "You can optionally add a custom prefix for the screenshot filenames.",
+          "placeholder": "Screenshot, CleanShot, etc."
         }
       ]
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.38.3"
+    "@raycast/api": "^1.73.2",
+    "trash-cli": "^5.0.0"
```

Comment:
Instead of using the CLI version, could we use the underlying `node` module instead ([`trash`](https://github.com/sindresorhus/trash)). I think we'd want to avoid spinning up another child process for a simple async command.

Reply:
Actually this was from an earlier attempt at this implementation, from what I can tell trash-cli is not necessary. I am using osascript to do the trash movement now so I will just remove this.